Source/santad/EventProviders/SNTEndpointSecurityProcessFileAccessAuthorizer.mm

```diff
@@ -56,6 +72,7 @@ - (void)enable {
 
   if (!self.isSubscribed) {
     if ([super subscribe:events]) {
+      LOGD(@"Proc FAA subscribed");
```

Comment:
Still needed?

Reply:
I like having these here for development, being debug it shouldn't matter too much and it's lightweight - it only prints when policies go from 0->1+ or 1+->0. Would like to keep them.

Reply:
That's fine, was just checking they weren't leftover debugging while writing